app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -107,4 +107,9 @@ object FeatureFlags {
      * Enables the wallpaper onboarding.
      */
     val wallpaperOnboardingEnabled = Config.channel.isDebug
+
+    /**
+     * Enables the wallpaper v2 enhancements.
+     */
+    val wallpaperV2Enabled = Config.channel.isDebug
```

Comment:
this flag will be short lived, but debating this verbiage vs something like `wallpaperEnhancementsEnabled`, if anyone has any strong opinions

Reply:
I think tying to language to the epic makes sense